aspnetcore/blazor/javascript-interoperability/static-server-rendering.md

```diff
@@ -165,7 +165,11 @@ In the RCL, add the following `PageScript` component.
 
 The `PageScript` component functions normally on the top-level of a page.
 
-If you place the `PageScript` component in the app's layout (for example, `Components/Layout/MainLayout.razor`), which results in a shared `PageScript` among pages that use the layout, then the component only runs `onLoad` after a full page reload and `onUpdate` when any enhanced page update occurs, including enhanced navigation.
+If you place the `PageScript` component in an app's layout (for example, `MainLayout.razor`), which results in a shared `PageScript` among pages that use the layout, then the component only runs `onLoad` after a full page reload and `onUpdate` when any enhanced page update occurs, including enhanced navigation:
+
+```razor
+<PageScript Src="./Components/Pages/PageWithScript.razor.js" />
```

Comment:
Sure ... do you have a suggestion?

Reply:
Thinking about this more, I'm not sure if having the code snippet adds much value. Maybe it's fine to remove it? We could alternatively come up with a name for a script containing "global" JS logic but maybe that raises more questions than it answers (where is the script defined, what does it do?).

Reply:
> the code snippet ... remove it?

I'm not sure which one you're referring to. They all look required to me given the text that's here.

Reply:
This one:

> If you place the `PageScript` component in the app's layout...
> ```razor
> <PageScript Src="./Components/Pages/PageWithScript.razor.js" />
> ```

The paragraph above the snippet talks about placing a `<PageScript>` in the app's layout, but then the snippet shows an unremarkable usage of `<PageScript>` without context showing where it's being rendered from. I'm guessing it's meant to be assumed that the `<PageScript>` in the snippet is placed somewhere in the layout (even though it's not explicitly shown in the snippet), but if that's the case, then what does the snippet show that the reader couldn't have already inferred?

Or maybe I'm just totally misinterpreting this section. Was the snippet intended to refer to the paragraph that specifically mentions placing the `<PageScript>` in the layout? Or is it supposed to show a general usage of `<PageScript>`? If it's the latter, that makes a lot more sense to me, although I'd maybe suggest moving the code snippet to be _above_ the paragraph that explains the nuance about placing a `<PageScript>` in the layout. And then we wouldn't need to rename the JS file because now the example clearly demonstrates the most common usage, which is to have `<PageScript>` be rendered from a page component.

Let me know if I'm still missing something! I think I was just wrongly associating the snippet with the comment about placing the component in the app's layout.